config/routes.rb

```diff
@@ -281,7 +281,7 @@
 
     resources :statements, controller: "facility_statements", only: [:index, :new, :show] do
```

Comment:
I believe we can add `create` to the `only` list here. That will change the path to be `POST /facilities/:facility_id/facility_statements` rather than `/create`.